schemas/session-replay/browser/mouse-interaction-schema.json

```diff
@@ -4,25 +4,44 @@
   "title": "MouseInteraction",
   "type": "object",
   "description": "Browser-specific. Schema of a MouseInteraction.",
-  "required": ["type", "id", "x", "y"],
-  "properties": {
-    "type": {
-      "type": "integer",
-      "enum": [0, 1, 2, 3, 4, 5, 6, 7, 9],
-      "description": "The type of MouseInteraction.",
-      "readOnly": true
+  "oneOf": [
+    {
+      "required": ["type", "id", "x", "y"],
+      "properties": {
+        "type": {
+          "type": "integer",
+          "enum": [0, 1, 2, 3, 4, 7, 9],
```

Comment:
Maybe it could be helpful to have in the description the meaning for each value. Apparently 5 and 6 means blur and focus, but what about the others?

Reply:
added to the description